.circleci/config.yml

```diff
@@ -0,0 +1,79 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/desktop/desktop
+    parallelism: 1
+    shell: /bin/bash --login
+    environment:
+      CIRCLE_ARTIFACTS: /tmp/circleci-artifacts
+      CIRCLE_TEST_REPORTS: /tmp/circleci-test-results
+    macos:
+      xcode: '9.3.0'
+    steps:
+    - checkout
+    - run: mkdir -p $CIRCLE_ARTIFACTS $CIRCLE_TEST_REPORTS
+    - restore_cache:
+        keys:
+        # when lock file changes, use increasingly general patterns to restore cache
+        - yarn-packages-v1-{{ .Branch }}-{{ checksum "yarn.lock" }}
```

Comment:
Chiming in here. Would you want to also add `-{{ checksum "app/yarn.lock" }}` and then also add `app/node_modules` to the cache?

Reply:
Thanks for the suggestion. The `postinstall` step will update the `app` with a `yarn install --force`, so caching here is not useful. I've opened #5318 to look into what using an offline cache might look like, and whether we can get it working faster than our current setup.